src/main.py

```diff
@@ -93,7 +104,24 @@ def run_plan_bot(
     - Reply to any unreplied matching comments (If replies are on)
     - Update replied_to list (If replies and tracking is on)
     """
-    print("Running a single pass of plan bot")
+    if log_level == "0":
+        level_name = logging.WARNING
+    elif log_level == "1":
+        level_name = logging.INFO
+    else:
+        level_name = logging.DEBUG
+
+    logging.basicConfig(
+        level=level_name, format="%(asctime)s %(levelname)s %(name)s: %(message)s"
```

Comment:
We should check what this looks like after it's up on GCP, to see how this format plays with Stackdriver (i.e. are we logging duplicate info)